2016-03-19 10:54:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
SELECT * FROM User WHERE name=? [42102-191]
2016-03-19 10:58:15 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 10:58:15 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 10:58:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 10:58:51 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 10:59:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 10:59:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(DriverManager.java:579)
	at java.sql.DriverManager.getConnection(DriverManager.java:221)
	at ir.visoft.accounting.db.Database.getConnection(Database.java:35)
	at ir.visoft.accounting.db.Database.test(Database.java:76)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 32 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:F:/repo/accounting/accounting/db/data/accounting.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 27 more
2016-03-19 11:00:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
SELECT * FROM User WHERE name=? [42102-191]
2016-03-19 11:01:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "USER" not found; SQL statement:
SELECT * FROM User WHERE name=? [42102-191]
2016-03-19 11:02:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: No data is available [2000-191]
2016-03-19 11:02:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "0" for parameter "columnIndex" [90008-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3172)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3200)
	at org.h2.jdbc.JdbcResultSet.getObject(JdbcResultSet.java:502)
	at ir.visoft.accounting.db.Database$1.handle(Database.java:53)
	at ir.visoft.accounting.db.Database$1.handle(Database.java:51)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:348)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:226)
	at ir.visoft.accounting.db.Database.test(Database.java:79)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
2016-03-19 11:02:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "0" for parameter "columnIndex" [90008-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:228)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3172)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3200)
	at org.h2.jdbc.JdbcResultSet.getObject(JdbcResultSet.java:502)
	at ir.visoft.accounting.db.Database$1.handle(Database.java:53)
	at ir.visoft.accounting.db.Database$1.handle(Database.java:51)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:348)
	at org.apache.commons.dbutils.QueryRunner.query(QueryRunner.java:226)
	at ir.visoft.accounting.db.Database.test(Database.java:79)
	at ir.visoft.accounting.Application.main(Application.java:16)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:293)
	at java.lang.Thread.run(Thread.java:722)
